interfaces/cython/cantera/examples/onedim/diffusion_flame_batch.py

```diff
@@ -76,8 +85,8 @@ def interrupt_extinction(t):
 
 # Exponents for the initial solution variation with changes in pressure Taken
 # from Fiala and Sattelmayer (2014). The exponents are adjusted such that the
-# strain rates increases proportional to p^(3/2), which results in flames
-# similar with respect to the extinction strain rate.
+# strain rates increases proportional to p^(3/2), which results in similar
+# flames with respect to the extinction strain rate.
```

Comment:
I think these lines should actually not be changed. In this case, the comment is using "similar" in a technical sense to mean that there is a similarity relationship between the conditions.